--- tools/build/src/tools/darwin.jam    2020-11-29 00:41:32.000000000 +0100
+++ tools/build/src/tools/darwin.jam    2020-11-29 00:42:34.000000000 +0100
@@ -502,6 +502,8 @@
         {
             if $(instruction-set) {
                 options = -arch$(_)$(instruction-set) ;
+            } else if $(address-model) = 64 {
+                options = -arch arm64 ;
             } else {
                 options = -arch arm ;
             }
