--- meson.build	2024-09-22 13:36:39
+++ meson.build	2024-09-22 13:37:02
@@ -198,10 +198,7 @@
   endif
 endif
 
-lzo_dep = dependency('lzo2', required: false)
-if lzo_dep.found()
-  conf.set('HAVE_LZO', 1)
-endif
+conf.set('HAVE_LZO', 0)
 
 dl_dep = cc.find_library('dl', required: false)
 if dl_dep.found() and cc.has_function('dlsym', dependencies: [dl_dep])
--- util/cairo-script/meson.build	2024-09-22 13:38:33
+++ util/cairo-script/meson.build	2024-09-22 13:38:59
@@ -27,7 +27,7 @@
 libcairoscript = library('cairo-script-interpreter',
   cairoscript_interpreter_sources,
   include_directories: [incbase],
-  dependencies: deps + [libcairo_dep, lzo_dep],
+  dependencies: deps + [libcairo_dep],
   soversion: cairo_version_sonum,
   version: cairo_libversion,
   c_args: ['-DCAIRO_COMPILATION'],
