--- pango/pangocoretext-fontmap.c	2024-09-22 16:26:09
+++ pango/pangocoretext-fontmap.c	2024-09-22 16:28:29
@@ -1641,13 +1641,6 @@
 G_DEFINE_TYPE (PangoCoreTextFontset,
                pango_core_text_fontset,
                PANGO_TYPE_FONTSET);
-
-#if !defined(MAC_OS_X_VERSION_10_8) || MAC_OS_X_VERSION_MIN_REQUIRED < MAC_OS_X_VERSION_10_8
-/* This symbol does exist in the CoreText library shipped with Snow
- * Leopard and Lion, however, it is not found in the public header files.
- */
-CFArrayRef CTFontCopyDefaultCascadeList (CTFontRef font_ref);
-#endif
 
 static PangoCoreTextFontset *
 pango_core_text_fontset_new (PangoCoreTextFontsetKey    *key,
@@ -1710,7 +1703,6 @@
   fontset->real_font_count = fontset->fonts->len;
 
   /* Add the cascade list for this language */
-#if defined(MAC_OS_X_VERSION_10_8) && MAC_OS_X_VERSION_MIN_REQUIRED >= MAC_OS_X_VERSION_10_8
     {
       CFArrayRef language_pref_list = NULL;
       CFStringRef languages[1];
@@ -1734,12 +1726,6 @@
           CFRelease (language_pref_list);
         }
     }
-#else
-  /* There is unfortunately no public API to retrieve the cascade list
-   * on Mac OS X < 10.8, so we use the following undocumented public function.
-   */
-  fontset->cascade_list = CTFontCopyDefaultCascadeList (pango_core_text_font_get_ctfont (best_font));
-#endif
 
   /* length of cascade list + real_font_count for the "real" fonts at the front */
   g_ptr_array_set_size (fontset->fonts, CFArrayGetCount (fontset->cascade_list) + fontset->real_font_count);
