
--- configure	Wed Jul 15 10:15:52 2020 -0700
+++ configure	Wed Jul 15 16:12:44 2020 -0700
@@ -21775,10 +21775,6 @@
           #import <Metal/Metal.h>
           #import <QuartzCore/CAMetalLayer.h>
 
-          #if !TARGET_CPU_X86_64
-          #error Metal doesn't work on this configuration
-          #endif
-
 int
 main ()
 {
@@ -24254,7 +24250,7 @@
     # so we'll just use libusb when it's available.
     case "$host" in
         # libusb does not support iOS
-        arm*-apple-darwin* | *-ios-* )
+        *-ios-* )
             skiplibusb=yes
             ;;
         # On the other hand, *BSD specifically uses libusb only
@@ -25154,7 +25150,7 @@
 fi
 
         ;;
-    arm*-apple-darwin*|*-ios-*)
+    *-ios-*)
         ARCH=ios
 
         CheckVisibilityHidden

--- configure.ac	Wed Jul 15 10:15:52 2020 -0700
+++ configure.ac	Wed Jul 15 16:12:44 2020 -0700
@@ -2156,10 +2156,6 @@
           #import <Cocoa/Cocoa.h>
           #import <Metal/Metal.h>
           #import <QuartzCore/CAMetalLayer.h>
-
-          #if !TARGET_CPU_X86_64
-          #error Metal doesn't work on this configuration
-          #endif
         ],[
         ],[
         have_metal=yes
@@ -3289,7 +3285,7 @@
     # so we'll just use libusb when it's available.
     case "$host" in
         # libusb does not support iOS
-        arm*-apple-darwin* | *-ios-* )
+        *-ios-* )
             skiplibusb=yes
             ;;
         # On the other hand, *BSD specifically uses libusb only
@@ -3872,7 +3868,7 @@
         #  other spins, like x86-64, use a more standard "libstdc++.so.*"
         AC_CHECK_FILE("/boot/system/lib/libstdc++.r4.so", EXTRA_LDFLAGS="$EXTRA_LDFLAGS -lstdc++.r4", EXTRA_LDFLAGS="$EXTRA_LDFLAGS -lstdc++")
         ;;
-    arm*-apple-darwin*|*-ios-*)
+    *-ios-*)
         ARCH=ios
 
         CheckVisibilityHidden
